package/gluon-logging/check_site.lua

```diff
@@ -0,0 +1,3 @@
+need_string({'syslog', 'ip'}, false)
```

Comment:
I haven't checked, but does this have to be an IP address or can it also be a hostname?

Reply:
According to https://openwrt.org/docs/guide-user/base-system/log.essentials#network_logging it's a "<Destination IP>".